platform/entanglement/src/actions/AbstractComposeAction.js

```diff
@@ -118,7 +118,6 @@ define(
                 currentParent = this.currentParent,
                 newParent = this.newParent,
                 object = this.object;
-
```

Comment:
@juliecwang - the following line deletions will make it harder to read (121 and 162). Just revert the changes on this particular file.